docs/source/reference/metrics.rst

```diff
@@ -181,3 +181,55 @@ Configure your InfluxDB and Telegraf InfluxDB output as usual, then on the Stats
 .. literalinclude:: /_includes/_telegraf_statsd_input.toml
     :language: toml
     
+Pushing metrics to Prometheus via the statsd_exporter
+=====================================================
+
+Prometheus provides a docker service called `statsd exporter <https://github.com/prometheus/statsd_exporter>` which receives data in the StatsD format and acts as a scrape target for Prometheus.
+
+This exporter enables to make the st2 metrics available to Prometheus and other monitoring solutions that are able to scrape Prometheus targets (i.e. Zabbix).
+
+While the README at https://github.com/prometheus/statsd_exporter provides the latest overview, metric mapping and service configuration https://github.com/prometheus/statsd_exporter#using-docker shows an example how to deploy the service using the prometheus/statds_exporter docker image. 
+
+The configuration examples below rely on the statsd_exporter default port configuration: 
+
+| port     | Purpose                                                   |
```

Comment:
This table doesn't render properly in `rst` for some reason.

Reply:
Addressed with the recent push. Used md instead of rst format for some reason, sorry.